source/systemUtils.py

```diff
@@ -63,6 +63,45 @@ def hasUiAccess():
 		ctypes.windll.kernel32.CloseHandle(token)
 
 
+TokenOrigin = 17
```

Comment:
Where does this value come from? Can you add a comment referencing relevant documentation if it exists.